styles/style.css

```diff
@@ -1,9 +1,19 @@
 /*Header styles*/
+@font-face {
+  font-family: "Roboto";
+  src: url("../about/fonts/Roboto-Medium.woff") format("woff");
+}
+@font-face {
+  font-family: "Roboto";
+  src: url("../about/fonts/Roboto-Regular.woff") format("woff");
+}
 .header {
   width: 1440px;
```

Comment:
Try not to use fixed proprties for continers.
You can apply code like this
.header {
  max-width: 1440px;
  width:100%;
  margin: 0 auto;
}

Reply:
This will let you write more flexable code.